index.html

```diff
@@ -259,6 +259,26 @@ <h6 class="description">Hello I’m a software developer! I can help you build a
       </section>
     </section>
   </div>
-       </section>
+</section>
+<footer class="form-cont">
+  <form action="https://formspree.io/f/mrgdkyla" method="post">
+    <h2 class="h2">I'm always<br>interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <input type="text" id="name" placeholder="Full name" required maxlength="30" name="full name"><div class="line"></div>
+    <input type="email" placeholder="Email address" required><div class="line" name="email"></div>
+    <textarea name="text" maxlength="500" required>Enter text here</textarea>
+    <button type="submit" class="get-in see1">Get in touch</button>
+  </form>
+  <div class="di"></div>
+  <section class="social">
+    <div class="git ico">
+      <img src="./images/icon2 twi.png" alt="">
+    </div>
+      <div class="lin ico"><img src="./images/icon2 li.png"></div>
+      <div class="ter ico"><img src="./images/angelist 1.png" alt=""></div>
+      <div class="twitter ico"><img src="./images/icontwi2.png" alt=""></div>
+      <div class="mess ico"><img src="./images/Vector.png" alt=""></div>
+    </section> 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As required for the project, kindly make sure, all the social icons are wrapped in a `ul` tag as their container, each item in a `li` tag, and each **social link** in a `<a>` tag for best semantics tags. Remember that all icons links should be used to access their specific social links